package.json

```diff
@@ -28,7 +28,7 @@
     "backbone": "1.2.3",
     "backbone-poller": "^1.1.3",
     "browserify-shim": "3.8.12",
-    "camshaft-reference": "0.4.0",
+    "camshaft-reference": "1.x || <1.x",
```

Comment:
1.x would mean `>=1.0.0 <2.0.0` which would exclude the current versions, a mayor version of 0 works differently than if >=1. I set it to this to have version range `>=0.0.0 <2.0.0`. See https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004 for more details